src/theme/theme.ts

```diff
@@ -264,6 +267,13 @@ const theme = createTheme({
       fontSize: "14px",
       lineHeight: "20px",
     },
+    body2: {
+      fontFamily: "Inter",
+      fontStyle: "normal",
+      fontWeight: 500,
```

Comment:
The only difference i see between body1 and body2 is the fontWeight, i believe you can still use the body1 in your case and override it like this:

```
<Typography variant="body1" sx={theme.typography.fontWeightMedium}>
  Enter text here
</Typography>

```